controller/rq_hover.ml

```diff
@@ -12,10 +12,28 @@ let build_ind_type mip = Inductive.type_of_inductive mip
 
 type id_info =
   | Notation of Pp.t
-  | Def of (Pp.t * Names.Constant.t option * string option)
-
-let info_of_ind env sigma ((sp, i) : Names.Ind.t) =
+  | Def of
+      { typ : Pp.t  (** type of the ide *)
+      ; params : Pp.t  (** params that need display next to the name *)
+      ; full_path : Names.Constant.t option
+            (** full path of the constant, if any, for example
+                [Stdlib.Lists.map] *)
+      ; file : string option  (** filename where the constant is located *)
```

Comment:
Do we have access to the location too? (The one goto definition uses). Not relevant for this PR, but it might be useful to include.

Reply:
We could, but not sure line number is relevant here, the client can indeed issue the corresponding specific request for location.